multisig/interaction/config/menu_functions.cfg

```diff
@@ -27,21 +27,39 @@ function upgrade-wrapper-chain-specific-token {
 
 function deploy-bridge-contracts {
   confirmation-with-skip deployAggregator
-  confirmation-with-skip stake
+  confirmation-with-skip manual-update-config-file
+  confirmation-with-skip stakeOracles
+
   confirmation-with-skip deployBridgedTokensWrapper
+  confirmation-with-skip manual-update-config-file
```

Comment:
So the just-deployed contracts are being reloaded so when we might need them on the init functions for the other contracts, the correct values will be passed. This will be completely refactored on v3.5.

Reply:
fixed